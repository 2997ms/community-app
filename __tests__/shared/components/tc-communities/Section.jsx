import React from 'react';
import Rnd from 'react-test-renderer/shallow';
import Section from 'components/tc-communities/Section';

const rnd = new Rnd();

test('Snapshot match', () => {
  rnd.render((
    <Section>
      <div>content</div>
    </Section>
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();

  rnd.render((
    <Section
      title="Section title"
      link={{
        title: 'link',
        url: '#',
      }}
      theme={{
        container: 'container',
        title: 'title',
        content: 'content',
        linkWrap: 'linkWrap',
        link: 'link',
      }}
    >
      <div>content</div>
    </Section>
  ));
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
