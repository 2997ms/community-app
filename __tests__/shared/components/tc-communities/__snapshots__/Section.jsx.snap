// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<Section
  link={null}
  theme={
    Object {
      "container": "src-shared-components-tc-communities-Section-___style__container___2DtNo",
      "content": "src-shared-components-tc-communities-Section-___style__content___17ZRh",
      "link": "src-shared-components-tc-communities-Section-___style__link___3nYY-",
      "linkWrap": "src-shared-components-tc-communities-Section-___style__linkWrap___1PnTb",
      "title": "src-shared-components-tc-communities-Section-___style__title___2uPtl",
    }
  }
  title={null}
>
  <div>
    content
  </div>
</Section>
`;

exports[`Snapshot match 2`] = `
<Section
  link={
    Object {
      "title": "link",
      "url": "#",
    }
  }
  theme={
    Object {
      "container": "src-shared-components-tc-communities-Section-___style__container___2DtNo container",
      "content": "src-shared-components-tc-communities-Section-___style__content___17ZRh content",
      "link": "src-shared-components-tc-communities-Section-___style__link___3nYY- link",
      "linkWrap": "src-shared-components-tc-communities-Section-___style__linkWrap___1PnTb linkWrap",
      "title": "src-shared-components-tc-communities-Section-___style__title___2uPtl title",
    }
  }
  title="Section title"
>
  <div>
    content
  </div>
</Section>
`;
